autoscale/src/main/java/com/sequenceiq/periscope/domain/ScalingTrigger.java

```diff
@@ -0,0 +1,118 @@
+package com.sequenceiq.periscope.domain;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import com.sequenceiq.periscope.api.model.TriggerStatus;
+
+@Entity
+@Table(name = "scaling_trigger")
```

Comment:
Maybe rename to something like ScalingStatus.
This is meant to be capturing multiple different scenarios - 1) Actual triggers and their status, 2) The inability to trigger an operation, 3) Cluster scaling being disabled as a result of the cluster being unhealthy (YARN/CM down)

Reply:
An enum called `ScalingStatus` already exists. Can this be called `ScalingOperation` instead?